js/index.js

```diff
@@ -1 +1,124 @@
 // Your code goes here
+
+
+//Focus event - focus, blur
+
+//CSS Animation Events
+
+//Mouse events
+
+ //mouseover
```

Comment:
1. mouseover, can you show how this is working for me? Dev tools doesn't like me very much